hls4ml/backends/vivado/passes/pointwise.py

```diff
@@ -1,23 +1,11 @@
 import numpy as np
-import re
 from copy import copy
 
 from hls4ml.model.optimizer import OptimizerPass
-from hls4ml.model.layers import Conv1D, Conv2D, register_layer
+from hls4ml.model.layers import register_layer
+from hls4ml.backends.fpga.fpga_layers import PointwiseConv1D, PointwiseConv2D
 from hls4ml.backends.vivado.passes.convolution_templates import Conv1DConfigTemplate, Conv1DFunctionTemplate, Conv2DConfigTemplate, Conv2DFunctionTemplate, conv1d_config_template, conv2d_config_template, conv_mult_config_template
 
-class PointwiseConv1D(Conv1D):
```

Comment:
This is actually a change in the Vivado backend. Is this class not needed? Can you provide more explanation?

Reply:
I see, it has been moved to fpga_layers. I think that solution makes sense, so I will resolve this.